src/sm.c

```diff
@@ -0,0 +1,480 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdlib.h>
```

Comment:
>de <s [](start = 6, length = 5)

I really like how this ended up looking!

Reply:
Thanks - it was right to push for change!

---
In reply to: [423421125](https://github.com/Azure/azure-c-util/pull/37#discussion_r423421125) [](ancestors = 423421125)